Widget/app/src/main/java/com/microsoft/device/display/samples/widget/WidgetFactory.kt

```diff
@@ -25,7 +26,7 @@ class WidgetFactory(private val context: Context, private val intent: Intent) :
     override fun onCreate() { }
 
     override fun onDataSetChanged() {
-        RssFeed.fetchRssFeed(context)
+        RssFeed.fetchRssFeed(context, RssSimpleApi::class.java)
```

Comment:
If we do what other comments do -> separate network configuration/usage from the real parser functionality we couldn't get rid of this change and the network stuff.

Reply:
I agree, the two functionalities are separated now